src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -616,11 +616,11 @@ public bool TryUnparse(StringBuilder sb)
 
         public abstract class ArgumentsBase
         {
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Comma separated list of stopwords", Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly)]
-            public string Stopwords;
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Comma separated list of stopwords", Name = "Stopwords", Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly)]
+            public string Stopword;
```

Comment:
>public [](start = 12, length = 6)

To be made internal in #2133, after #2059 gets merged

Reply:
Name is an alias (with my changes to the cmd line runner). I'm adding the Name attribute so that behavior of existing command lines doesn't get affected with renaming of the field name.